mixer/adapter/bypass/config/config.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// $title: Bypass
+// $description: Adapter for connecting to gRPC based remote adapters via the inline adapter model for a
+// fixed set of templates.
```

Comment:
Please combine the description onto a single line, otherwise the parser won't be happy.